src/components/sidebar/left-sidebar/TotalStats.tsx

```diff
@@ -24,36 +25,37 @@ export default function TotalStats() {
   const countries = countryDict.length;
 
   const countriesOrCountry = () => {

```

Comment:
I've seen a few of these general string manipulation functions pop up in our code. Would it be best to move them to some generic utils file?

Pretty minor and not at all necessary for this PR btw

Reply:
gonna leave this for now